examples/group_level/plot_spatiotemporal_tfce.py

```diff
@@ -0,0 +1,264 @@
+
+    # save results
+    betas_evoked[str(subjects[iteration])] = lm_betas
+
+    # clean up
+    del linear_model
+
+###############################################################################
+# compute mean beta-coefficient for predictor phase-coherence
+
+# subject ids
+subjects = [str(subj) for subj in subjects]
+
+# extract phase-coherence betas for each subject
+phase_coherence = [betas_evoked[subj]['phase-coherence'] for subj in subjects]
+
+# average phase-coherence betas
+weights = np.repeat(1 / len(phase_coherence), len(phase_coherence))
+ga_phase_coherence = combine_evoked(phase_coherence, weights=weights)
+
+###############################################################################
+# compute bootstrap confidence interval for phase-coherence betas and t-values
+
+# set random state for replication
+random_state = 42
+random = np.random.RandomState(random_state)
+
+# number of random samples
+boot = 2000
+
+# place holders for bootstrap samples
+cluster_H0 = np.zeros(boot)
+f_H0 = np.zeros(boot)
+
+# setup connectivity
+n_tests = betas.shape[1]
+connectivity, ch_names = find_ch_connectivity(epochs_info, ch_type='eeg')
+connectivity = _setup_connectivity(connectivity, n_tests, n_times)
+
+# threshold parameters for clustering
+threshold = dict(start=.1, step=.1)
+
+# run bootstrap for regression coefficients
+for i in range(boot):
+    # extract random subjects from overall sample
+    resampled_subjects = random.choice(range(betas.shape[0]),
+                                       betas.shape[0],
+                                       replace=True)
+    # resampled betas
+    resampled_betas = betas[resampled_subjects, :]
+
+    # compute standard error of bootstrap sample
+    se = resampled_betas.std(axis=0) / np.sqrt(resampled_betas.shape[0])
+
+    # center re-sampled betas around zero
+    for subj_ind in range(resampled_betas.shape[0]):
+        resampled_betas[subj_ind, :] = resampled_betas[subj_ind, :] - \
+                                       betas.mean(axis=0)
+
+    # compute t-values for bootstrap sample
+    t_val = resampled_betas.mean(axis=0) / se
+    # transfrom to f-values
+    f_vals = t_val ** 2
+
+    # transpose for clustering
+    f_vals = f_vals.reshape((n_channels, n_times))
+    f_vals = np.transpose(f_vals, (1, 0))
+    f_vals = f_vals.reshape((n_times * n_channels))
+
+    # compute clustering on squared t-values (i.e., f-values)
+    clusters, cluster_stats = _find_clusters(f_vals,
+                                             threshold=threshold,
+                                             connectivity=connectivity,
+                                             tail=1)
+    # save max cluster mass. Combined, the max cluster mass values from
+    # computed on the basis of the bootstrap samples provide an approximation
+    # of the cluster mass distribution under H0
+    if len(clusters):
+        cluster_H0[i] = cluster_stats.max()
+    else:
+        cluster_H0[i] = np.nan
```

Comment:
`cluster_H0[i] = cluster_stats.max() if len(clusters) else np.nan` is clearer IMO.

Reply:
ok, I'll keep the "explicit" for-loop for the purpose of readability (at least for now)